### ISSUE TITLE
AVT 1 - React DataTable w/Selection, DataTable w/dynamic, DataTable w/ boolean column & DataTable w/options content have DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3740]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP IBM Accessibility July 2019 Ruleset

## Detailed Description
Go to the React Datatable Component w/ Selection
Run DAP and the following issue is found on the copy button (see screenshot):
<img width="1287" alt="Screen Shot 2019-08-15 at 10 31 38 AM" src="https://user-images.githubusercontent.com/21676914/63106359-42c0e880-bf48-11e9-8410-cc8fd66d7d48.png">
<img width="1328" alt="Screen Shot 2019-08-15 at 10 31 24 AM" src="https://user-images.githubusercontent.com/21676914/63106360-448aac00-bf48-11e9-9e08-a3449c1a0ad5.png">

Note: The checkboxes are referencing empty labels which is causing the violations.  To fix the issue an aria-label is needed on the <input> or the <label> that is referenced needs to include descriptive text.

**Comment by fbarroso24** [comment]
The following issue, https://github.ibm.com/IBMa/accessibility-web-engine/issues/214, was recently opened for the `DAP` team to investigate to see if this is a `False-Positive` violation.

**Comment by fbarroso24** [comment]
Hi @dakahn, I think you can resolve this as working as designed.  As you know I had been looking into fixing the issue because we had also run into this in our product as a result of leveraging this component.  After discussing this on the `Carbon-React` channel https://ibmmasslab.slack.com/archives/C2K6RFJ1G/p1573581064225600 and then getting input from the `Accessibility-at-ibm` community https://ibmmasslab.slack.com/archives/C036P1CTN/p1574285329429200   a `DAP` issue was opened to investigate these violations https://github.ibm.com/IBMa/accessibility-web-engine/issues/214 and an update has been made to their `Rules` that will no longer flag these 2 things as `Violations`.  You can update your `Archive` in your `DAP` tool and test it out yourself today if you'd like. For instructions on how to do that see this comment.  https://github.ibm.com/IBMa/accessibility-web-engine/issues/214#issuecomment-16154068

As of right now, it has been fixed in the `Preview` archive.  However, it will soon be included in `DAP`'s next release of their ruleset as mentioned in the `DAP` issue.    Let me know if you have any questions.  Thanks.

**Comment by asudoh** [comment]
Thanks @fbarroso24 for your detailed analysis! The DAP errors in the issue description seems no longer producible with latest `master` with November 2019 ruleset and the link you referred to confirms such. Closing.
